Dockerfile

```diff
@@ -28,9 +28,9 @@ FROM ubuntu:22.04
 RUN mkdir -p /trin/trin-core/src/assets
 COPY --from=builder /trin/trin-core/src/assets/merge_macc.bin ./trin/trin-core/src/assets/merge_macc.bin
 # copy build artifacts from build stage
-COPY --from=builder /trin/target/release/trin .
-COPY --from=builder /trin/target/release/trin-cli .
+COPY --from=builder /trin/target/release/trin /usr/bin/
+COPY --from=builder /trin/target/release/trin-cli /usr/bin/
 
 ENV RUST_LOG=debug
 
-ENTRYPOINT ["./trin"]
```

Comment:
Did you read my mind? Haha, I've been dealing with this in my docker image for running trin with a bridge node. Since I use the trin image as the base for the bridge image. But, in my case this pr doesn't help due to ... https://github.com/carver/eth-portal/blob/main/eth_portal/trin.py#L32 ... where it's hardcoded how to launch the executable as `./trin`.

Reply:
Of course, we could change how it's hardcoded in `eth-portal`...

Reply:
This issue also breaks `portal-hive` because I can't run trin with `./trin` after we changed it from `./trin-main`. Fluffy also store its binary file in `/usr/bin` which I think is the correct way to deal with it.